### PR TITLE
Feature/descw 1652 wp documentation

### PR DESCRIPTION
**Pull request by ShawnTurple**

### CODE REVIEW

**Comment by codewisenate** [comment]
Is the intent to have this run under the Appearance > Using the BCGov Block Theme? eg: 
image

I see a: Cannot modify header information - headers already sent PHP warning when viewing that page in the admin. I am able to run the documentation VuePress locally. eg:
image

File: documentation/docs/guide/content/Block-Patterns.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+Patterns are separated into different categories such as General, Banners, Header/Footer, Page Layout, Post Queries, etc. These categories are used to separate intended purpose and also keep the list of patterns from getting too large and inhibit browsing. Should you know the name of a pattern you can use the search tool in the Block Inserter to filter and find the one you need.
+
+Many patterns are available for use as part of the default BCGov base theme. They offer layouts such as Alternating Cards, Card with Image Overlay, Card with Under Image Portrait 2-Up, Cards Portrait 3-up, Detail Card with Icons, Hero Banner, Long Card, Quote, Small Quote with Image, and many more.
+
+## Creating Custom Patterns
+
+<img src="/bcgov-wordpress-block-theme/patterns-admin-menu.png" align="right" height="175" hspace="15">
+
+Enabling printing of an imageIn the site admin click on "Patterns", this will bring up the list of custom patterns currently in your site, or it will be empty if starting new.
```
Comment (mhaswell-bcgov) [inline documentation/docs/guide/content/Block-Patterns.md]: Looks like some formatting was lost here.

File: documentation/docs/guide/content/Block-Patterns.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+Once you've arranged the blocks to your liking, click on the "Publish" button to save your pattern.
+
+## Adding custom patterns to the pattern chooser
+
+
+To add your custom pattern to the Pattern chooser, you need to add a new or existing "Pattern Group" located on the Custom Pattern tab of the block inspector area. Click on the "Add New Pattern Group" link or choose from any of the options listed.
+
+<img src="/bcgov-wordpress-block-theme/pattern-settings.png" align="right" height="300" hspace="15">
+
+Enabling printing of an imageIf creating a new group enter a "New Pattern Group Name" for your new pattern group and select the group option you just created.
```
Comment (mhaswell-bcgov) [inline documentation/docs/guide/content/Block-Patterns.md]: Formatting issue here too.

File: documentation/docs/.vuepress/config.js [hunk 3]
```diff
@@ -1,18 +1,53 @@
 import { defaultTheme } from "@vuepress/theme-default";
 import { defineUserConfig } from "vuepress";
+import { searchPlugin } from "@vuepress/plugin-search";
+import { mdEnhancePlugin } from "vuepress-plugin-md-enhance";
+import { getDirname, path } from '@vuepress/utils'
 
+console.error(__dirname)
```
Comment (mhaswell-bcgov) [inline documentation/docs/.vuepress/config.js]:
```suggestion
```
I'm assuming we don't need this.

**Comment by mhaswell-bcgov** [comment]
> I see a: Cannot modify header information - headers already sent PHP warning when viewing that page in the admin.

I believe this is caused by the AIOSEO plugin. I don't think it was introduced in this PR.

File: documentation/docs/guide/content/Block-Patterns.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+Patterns are separated into different categories such as General, Banners, Header/Footer, Page Layout, Post Queries, etc. These categories are used to separate intended purpose and also keep the list of patterns from getting too large and inhibit browsing. Should you know the name of a pattern you can use the search tool in the Block Inserter to filter and find the one you need.
+
+Many patterns are available for use as part of the default BCGov base theme. They offer layouts such as Alternating Cards, Card with Image Overlay, Card with Under Image Portrait 2-Up, Cards Portrait 3-up, Detail Card with Icons, Hero Banner, Long Card, Quote, Small Quote with Image, and many more.
+
+## Creating Custom Patterns
+
+<img src="/bcgov-wordpress-block-theme/patterns-admin-menu.png" align="right" height="175" hspace="15">
+
+## Enabling printing of an image
```
Comment (mhaswell-bcgov) [inline documentation/docs/guide/content/Block-Patterns.md:29]: Actually now that I read these headings, they don't describe what is underneath them so I guess they should just be deleted.

**Review by mhaswell-bcgov — APPROVED** [review]

**Pull request merged**